### ISSUE TITLE
Application Insights Kusto queries do not support $__timeFrom() and $__timeTo() despite documentation instructions

### ISSUE THREAD

**Issue by christianrondeau** [grafana/grafana#21208]
Documentation states that we can use `$__timeFrom()` in Azure Application Insights (Azure Monitor) data source custom queries (e.g. using Application Insights, use the text editor, i.e. the small pen icon in the query editor, and try using variables).

`$__timeFilter()` works, but using `$__timeFrom()` and `$__timeTo()` yields a "Metric request error" error. Also, `$__from` and `$__to` work (they provide an epoch value) but they do not give the same values as `$__timeFilter()` (probably due to time zones), which makes `$__timeFrom()` necessary for `make-series` queries.

To reproduce, use this query, which should work in any Application Insights account:

```
requests 
| take 1
| project timestamp, splitOn = "$__timeFrom()", y = $__from
```

**Environment**:
- Grafana version: Grafana v6.5.2 (742d165) (Cloud)
- Data source type & version: Azure Monitor
- OS Grafana is installed on: Cloud
- User OS & Browser: Windows 10, Chrome

**Comment by christianrondeau** [comment]
Also if someone is in need of a workaround, here's how I'm getting the from/to values:

```
let timeRange = requests | where $__timeFilter(timestamp) | summarize min = min(timestamp), max = max(timestamp);
let timeFrom = toscalar(timeRange | project min);
let timeTo = toscalar(timeRange | project max);
myRequest...
| make-series ... in range(timeFrom, timeTo, $__interval)
```
```

**Comment by daniellee** [comment]
Fixed by #21879
